articles/native-platforms/ios-swift/09-mfa.md

```diff
@@ -0,0 +1,30 @@
+---
+title: MFA
+description: This tutorial will show you how to configure Multi Factor Authentication (MFA) via Google Authenticator in your app.
+---
+
+### 1. Enable MFA in your account
+
+You have to enable the MFA feature in your account. 
+
+First, follow [this link](https://manage.auth0.com/#/multifactor). Then, turn the switch on for **Google Authenticator**, under the *Choose a Provider* section.
```

Comment:
@pablolvillar replace link `[this link](https://manage.auth0.com/#/multifactor)` with `[this link](${uiURL}/#/multifactor)`

https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables